team-ONEBILLION/video-events/video-events-ONEBILLION_2019-03-01.csv

```diff
@@ -0,0 +1,3840 @@
+tablet_serial,video_id,start_time,end_time
+5A23002905,5054,1542187510,1542187647
+5A23002905,5054,1542188433,1542188553
+5A23002905,5054,1542188558,1542188677
+5A23002905,5054,1542192106,1542192225
+5A23002905,5054,1542193913,1542194031
+5A23002905,5054,1542194043,1542194162
+5A23002905,5054,1542194714,0

```

Comment:
@XPRIZE/glexp-team-onebillion What does it mean when `end_time` equals zero? That the child turned off the tablet screen without watching the video until the end?